UnitTests/BraintreeAmericanExpressTests/BTAmericanExpressClient_Tests.swift

```diff
@@ -12,7 +12,20 @@ class BTAmericanExpressClient_Tests: XCTestCase {
         super.setUp()
         mockAPIClient = MockAPIClient(authorization: "development_tokenization_key")!
         amexClient = BTAmericanExpressClient(apiClient: mockAPIClient)
-   }
+    }
+    
+    func testGetRewardsBalance_formatsGETRequest() async {
+        let result = try? await amexClient!.getRewardsBalance(forNonce: "fake-nonce", currencyISOCode: "fake-code")
+        
+        XCTAssertEqual(mockAPIClient.lastGETPath, "v1/payment_methods/amex_rewards_balance")
+        
+        guard let lastGetParameters = mockAPIClient.lastGETParameters else {
+            XCTFail("Expected GET parameters")
+            return
+        }
+        XCTAssertEqual(lastGetParameters["currencyIsoCode"] as! String, "fake-code")
+        XCTAssertEqual(lastGetParameters["paymentMethodNonce"] as! String, "fake-nonce")
```

Comment:
This work has been a good exercise, since it reveals that we aren't testing the formatting of our JSON bodies for both GET & POST requests at all right now 😬 (as also evidenced in https://github.com/braintree/braintree_ios/pull/1151)

Reply:
I didn't completely understand the example that reveals we aren't testing the formatting of JSON bodies. Maybe understanding it would give me the answer to my question.

Q: Why do we care about the _formatting_ of the JSON as long as the key value pairs are the same?